content/scout/policy.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Get started with Policy Evaluation in Docker Scout
+keywords: scout, supply chain, vulnerabilities, packages, cves, policy
+description: |
+  Policies in Docker Scout let you define supply chain rules and thresholds
+  for your artifacts, and track how your artifacts perform against those
+  requirements over time
+---
+
+> **Beta**
+>
+> Policy Evaluation is a [Beta](/release-lifecycle/#beta) feature of Docker
+> Scout. This feature is available to organizations participating in the
+> limited preview program for policies.
+>
+> If you're interested in trying out this feature, reach out using the form on
+> the [Docker Scout product page](https://docker.com/products/docker-scout)
+{ .restricted }
+
+In software supply chain management, maintaining the security and reliability
+of artifacts is a top priority. Policy Evaluation in Docker Scout introduces a
+layer of control, on top of existing analysis capabilities. It lets you define
+supply chain rules for your artifacts, and helps you track how your artifacts
+perform, relative to your rules and thresholds, over time.
+
+Learn how you can use Policy Evaluation to ensure that your artifacts align
+with established best practices.
+
+## How it works
+
+When you activate Docker Scout for a repository, images that you push are
+[automatically analyzed](../advanced-image-analysis.md). The analysis gives you
+insights about the composition of your images, including what packages they
+contain, and what vulnerabilities they're exposed to. Policy Evaluation builds
+on top of the image analysis feature, interpreting the analysis results against
+the rules defined by policies.
+
+A policy defines one or more criteria that your artifacts should fulfill. For
+example, one of the default policies in Docker Scout is the **Critical
+vulnerabilities** policy, which proclaims that your artifacts must not contain
+any critical vulnerabilities. If an artifact contains one or more
+vulnerabilities with a critical severity, that artifact fails the evaluation.
+
+In Docker Scout, policies are designed to help you ratchet forward your
```

Comment:
Wondering if we can use a simpler word instead of 'ratchet forward'? Maybe improve, enhance?